Source/ACE/Entity/MovementData.cs

```diff
@@ -250,6 +169,29 @@ public MovementData ConvertToClientAccepted(uint holdKey, CreatureSkill run)
             return md;
         }
 
+        public void SetMovementStateFlag()
+        {
```

Comment:
Flags are only used when we serialize - I can move it to the serialize method and call it there?